content/virtualization.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Virtualization on Linux
+description: Create a VM on Linux with GNOME Boxes or Virtual Machine Manager
+keywords:
+  - VM
+  - libvirt
+  - GNOME Boxes
+  - virt-manager
+  - Virtual Machine Manager
+  - VMM
+  - VirtualBox
+  - VMWare
+  - Virtual Machine
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+![GNOME Boxes with VMs](/images/gnome-boxes/boxes-full.png)
```

Comment:
Minor aesthetics - this screenshot is massive with a lot of blank gray space. The text is difficult to read. Is it possible to crop?
![image](https://user-images.githubusercontent.com/80931038/159061534-49c695a5-9226-4f11-b47c-be2dbf50e0c0.png)

Reply:
The screenshot was to show what the Boxes interface looks like with VMs installed. There is no way to increase the font size in the app as far as I can see.

But, the text is just the names of operating systems, and is not important to the content.

Reply:
Can you re-take the screenshot after resizing the window vertically so there's not so much empty space at the bottom? I would go about 1/3 of the current height, or leave enough room at the bottom that it's clear the list is over.

Reply:
This is just a heads-up, but in Pop!_OS 22.04 pre-release, GNOME Boxes appears to be using libadwaita, as its interface has the Adwaita Dark color scheme instead of Pop Dark.

The screenshot doesn't quite look like Pop Dark, is it using a third-party theme?

Reply:
@Ampersandstorm I can certainly crop the photo, but I was wondering what your thoughts were with the above information. What do you think would work better for this article?

Reply:
Sorry, did not see @jacobgkau's messages when I responded. I'll redo the screen shots with the default Pop theming.

Reply:
@gangwerz Thanks! Off-topic but I'm wondering what screenshot program you use to apply the slight shadow effect on the edges?

Reply:
@Ampersandstorm I was using the built-in GNOME Screenshot tool for all the pictures. I'm not sure if the shadow is because the photos are PNGs, or if it's due to CSS settings on this page